README.md

```diff
@@ -1,3 +1,80 @@
-# The Nivenly Covenant
+# The Nivenly Covenant 1.0
+
+Include a published and publicly accessible statement you to agree and commit to the following terms. 
+
+``` 
+<Entity,Person(s),Authors,Organization(s)> herby agree an unequivocally commit to the fullest
+extent of our understanding, our intentions, and our goals to The Nivenly Covenant 1.0
+```
+
+### 1. Sustainability
+
+We agree that sustainability is our critical priority, and is necessary for the longevity of future generations of stewards and technologists.
```

Comment:
can we define sustainability or is the determination of the project's adherence to this something for a Tech Advisory Group to determine?